jsamchineme/send-it
# Add postman collections to readme 161920255

#### What does this PR do?
- Make available on the readme, a collection of all API endpoints so far
#### Description of Task to be completed?
- Update the readme with the postman collection
#### How should this be manually tested?
- https://github.com/jsamchineme/send-it/tree/ch-add-postman-collections-to-readme-161920255#installation 
- open the link copy the text and import to postman as raw text
- Save and check out the end points
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
- https://www.pivotaltracker.com/story/show/161920255
#### Screenshots (if appropriate)
None
#### Questions:`
None
